lib/stat_tracker.rb

```diff
@@ -143,14 +143,14 @@ def games_by_season
     end
   end
 
-  def preseason_games
-    @games.find_all do |game|
+  def preseason_games(games = @games)
```

Comment:
gave this a default value so that i could pass a different set of games to it. I passed the games grouped by season here, so that then I could pull only the preseason games